### ISSUE TITLE
Unable to download/open sent and received images in Skype (URL handling)

### ISSUE THREAD

**Issue by MJB17** [meetfranz/recipe-skype#2]
I'm having trouble opening images that I receive or send through Skype. The thumbnail shows fine, but when I click it to open the image, Franz opens a browser window and links directly to a file called 'imgpsh_fullsize.dms', which is downloaded instantly but can't be opened after downloading. Anybody else experiencing this issue? 

(Using Franz 5.0.0-beta.10 on macOS Sierra 10.12.6 and Safari 11.0)

<img width="290" alt="screen shot 2017-10-19 at 09 33 39" src="https://user-images.githubusercontent.com/17046072/31765857-209596cc-b4c6-11e7-9d32-0e29de7ce12f.png">

**Comment by Wouter0100** [comment]
I'm not able to open any images indeed. Will try to fix this and open a PR :+1:

**Comment by ritandrade** [comment]
Same here. except it doesn't download any file, it just sends me to an blank URL. I'm on Windows 10 PRO, using Franz 5.0.0-beta.11 and my default browser is chrome 61.0.3163.100

**Comment by Wouter0100** [comment]
I've found out, when clicking on a image and it opens a blank URL in the browser is fixed by logging into Skype in that browser. Then it'll load the image as a temporary workaround.

**Comment by adlk** [comment]
Seems like my fix was not good for all types of links... 😒

**Comment by Kwadz** [comment]
Yes, I have a similar issue here. When I click on a picture received in Skype, Franz opens my favorite browser and the page is not authorised.

Example: https://api.asm.skype.com/v1/objects/0-weu-d10-61b4e054eee94e55e07f1d74057bec95/views/imgpsh_fullsize

[
<img width="926" alt="screen shot 2018-02-16 at 14 55 08" src="https://user-images.githubusercontent.com/1472599/36310913-c8e738dc-1329-11e8-88af-9079938dc853.png">
](url)

**Comment by dmknght** [comment]
Same here. Click on image directs me to web browser, and browser gives me Unauthenticated response. Download file crashed franz. I am on Debian 9

**Comment by 4b3ldu4rt3** [comment]
same here

**Comment by kermitthegreat** [comment]
same here :)

**Comment by marcolong** [comment]
any news?

**Comment by SeverusWu** [comment]
Please go to http://web.skype.com to login your Skype account first. And remember to click "keep login". Then you can close the browser and go back to Franz to click the image you want to download.

The reason is browser needs your authorization...

**Comment by MarioAzulay** [comment]
it's 2019 already ... still no solution to this problem ?

**Comment by calinpristavu** [comment]
> it's 2019 already ... still no solution to this problem ?

@MarioAzulay The solution is posted literally 1 comment before yours by @SeverusWu .
Just log in to web skype, tick "keep me logged in", and then you can download stuff in meetfranz skype

**Comment by MarioAzulay** [comment]
@calinpristavu that's not a solution, its a "work-round-kinda-fix"! a solution would be a permanent fix to this problem.

**Comment by gramoflava** [comment]
I would like a solution as well. The workaround is still just that — a workaround. Web Skype isn't even working in Safari. I understand that I must switch now also to Chrome, correct?
In comparison, Rambox recommends to disable previews and then files can be downloaded. Doesn't help Franz though.

**Comment by xross** [comment]
Same issue on OSX. Files downloaded as 'original.dms'

**Comment by Mirta778** [comment]
> Same issue on OSX. Files downloaded as 'original.dms'

Yes
